### ISSUE TITLE
npm audit vunerablities due to outdated dependencies

### ISSUE THREAD

**Issue by Cherviakov** [Ziv-Barber/officegen#297]
```
                       === npm audit security report ===                        
                                                                                
┌──────────────────────────────────────────────────────────────────────────────┐
│                                Manual Review                                 │
│            Some vulnerabilities require your attention to resolve            │
│                                                                              │
│         Visit https://go.npm.me/audit-guide for additional guidance          │
└──────────────────────────────────────────────────────────────────────────────┘
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ Moderate      │ Prototype Pollution                                          │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ lodash                                                       │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Patched in    │ >=4.17.11                                                    │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ officegen                                                    │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ officegen > xmlbuilder > lodash                              │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://nodesecurity.io/advisories/782                       │
└───────────────┴──────────────────────────────────────────────────────────────┘
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ Low           │ Prototype Pollution                                          │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ lodash                                                       │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Patched in    │ >=4.17.5                                                     │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ officegen                                                    │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ officegen > xmlbuilder > lodash                              │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://nodesecurity.io/advisories/577                       │
└───────────────┴──────────────────────────────────────────────────────────────┘
```

**Comment by askoufis** [comment]
There's also this one too https://snyk.io/vuln/SNYK-JS-LODASH-73639

**Comment by Ziv-Barber** [comment]
I'll add it to the list of things to work on.
I'm not sure that all of the dependencies update will be ready for the next release.

**Comment by Ziv-Barber** [comment]
OK, almost finished the xmlbuilder migrating so it'll be ready for the next release.

**Comment by Ziv-Barber** [comment]
Fixed on 0.4.8
